erpnext/buying/doctype/request_for_quotation/request_for_quotation.js

```diff
@@ -124,6 +124,14 @@ frappe.ui.form.on("Request for Quotation",{
 		dialog.show()
 	},
 
+	required_date(frm) {
+		if (frm.doc.required_date) {
+			$.each(frm.doc.items || [], function(i, d) {
+				d.schedule_date = frm.doc.required_date;
```

Comment:
only change it if it is not already set? (this is how it works in other doctypes too IIRC)

Also, use ES6 instead of jquery.

Reply:
@noahjacob update on this?